city_scrapers/spiders/il_aging_advisory_council.py

```diff
@@ -0,0 +1,119 @@
+        needs.
+        """
+        self._validate_location(response)
+
+        self._validate_meeting_times(response)
+
+        table = response.xpath(
+            '//*[@id="ctl00_PlaceHolderMain_ctl01__ControlWrapper_RichHtmlField"]\
+            /table'
+        )
+
+        for table_row in table.xpath('.//tr'):
+            row_date = table_row.xpath('.//strong/\
+            text()').re(r'([A-Z]\w+)\s*(\d\d?),*\s*(\d\d\d\d)')
+            if not row_date:
+                continue
+            else:
+                converted_date = self._convert_date(row_date)
+
+            row_urls = table_row.xpath('.//a')
+
+            meeting = Meeting(
+                title='Illinois Department on Aging Advisory Committee Meetings',
+                description='',
+                classification=ADVISORY_COMMITTEE,
+                start=self._parse_start(converted_date),
+                end=self._parse_end(converted_date),
+                all_day=self._parse_all_day(table_row),
+                time_notes='',
+                location=self.location,
+                links=self._parse_links(row_urls),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _validate_location(self, response):
+        location_test = response.xpath(
+            '//*[@id="ctl00_PlaceHolderMain_ctl01__ControlWrapper_RichHtmlField"]\
+        /ul/li[2]/text()'
+        ).get()
+        if '7th Floor, 160 N. LaSalle Street' not in location_test:
+            raise ValueError("Meeting location has changed")
+        else:
+            pass
+
+    def _validate_meeting_times(self, response):
+        meeting_time = response.xpath(
+            '//*[@id="ctl00_PlaceHolderMain_ctl01__ControlWrapper_RichHtmlField"]\
+        /p[1]/text()'
+        ).get()
+        if '1 p.m. - 3 p.m' not in meeting_time:
+            raise ValueError("Meeting times have changed")
+
+    def _convert_date(self, item):
+        """Parse start datetime as a naive datetime object."""
+        month = item[0]
+        day = item[1]
+        year = item[2]
+        converted_date = datetime.strptime(month[0:3] + day + year, '%b%d%Y')
+        return converted_date
+
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+        start_time = item.replace(hour=13)
+        return start_time
+
+    def _parse_end(self, item):
+        """Parse end datetime as a naive datetime object. Added by pipeline if None"""
+        end_time = item.replace(hour=15)
+        return end_time
+
+    def _parse_links(self, item):
+        """Parse or generate links."""
+        links = []
+        for i in item:
+            href = i.xpath('@href').get()
+            full_href = 'https://www2.illinois.gov' + href
+            text = i.xpath('text()').get()
```

Comment:
We'll want to call `.strip()` to remove leading or trailing whitespace on the title and `href` here to keep the output clean. There are some extra whitespace characters in the output now: https://github.com/City-Bureau/city-scrapers/runs/392391279#step:11:79

Also, we'll need to check if the value is `None` before stripping the whitespace, since it looks like that's the case for this link: https://github.com/City-Bureau/city-scrapers/runs/392391279#step:11:144 If the value for title or `href` is `None` anywhere we should ignore that link and `continue` to the next item